### PR TITLE
[CSS-In-HTML] Small style changes

### PR DESCRIPTION
**Pull request by Drake-Eidukas**
Currently, there are small refactors required for some more obtuse code styling.
This change simply refactors some particularly egregious style to be more readable.

Also learning how to make proper pull requests 👍

### CODE REVIEW

**Comment by Drake-Eidukas** [comment]
:)

**Comment by pranaygp** [comment]
Also, with GitHub issues/PRs, if you're referring to another issue/PR, it's always good to make an explicit reference like #20. That way, a reference back to this issue/PR also shows up on that reference issue. Also, a cool trick is having the like "Closes #xx" at the end of your first PR message means that when this PR gets merge in, the target issue is automatically closed (same is true with commits)

File: server/src/core/findSelector.ts [hunk 1]
```diff
@@ -22,11 +22,15 @@ export default function findSelector(document: TextDocument, position: Position)
   let start = offset;
   let end = offset;
 
-  while (start > 0 && text.charAt(start - 1) !== ' ' && text.charAt(start - 1) !== '\'' && text.charAt(start - 1) !== '"' && text.charAt(start - 1) !== '\n' && text.charAt(start - 1) !== '<')
-    start -= 1
+  let delimeters = [' ', '\'', '"', '\n', '<']
```
Comment (pranaygp) [inline server/src/core/findSelector.ts]:
```javascript
const delimeters = [' ', '\'', '"', '\n', '<'];
```

File: server/src/core/findSelector.ts [hunk 2]
```diff
@@ -22,11 +22,15 @@ export default function findSelector(document: TextDocument, position: Position)
...
 
-  while (end < text.length && text.charAt(end) !== ' ' && text.charAt(end) !== '\'' && text.charAt(end) !== '"' && text.charAt(end) !== '\n' && text.charAt(end) !== '>')
+  while (start > 0 && delimeters.indexOf(text.charAt(start - 1)) === -1) {
```
Comment (pranaygp) [inline server/src/core/findSelector.ts]: how about `!delimeters.includes(text.charAt(start - 1))`?

File: server/src/core/findSelector.ts [hunk 3]
```diff
@@ -22,11 +22,15 @@ export default function findSelector(document: TextDocument, position: Position)
...
+    start -= 1
+  }
+  
+  while (end < text.length && delimeters.indexOf(text.charAt(end)) === -1) {
```
Comment (pranaygp) [inline server/src/core/findSelector.ts]: same as above

File: server/src/core/findSelector.ts [hunk 4]
```diff
@@ -47,21 +51,20 @@ export default function findSelector(document: TextDocument, position: Position)
...
           }
         }
         break;
     }
     tokenType = htmlScanner.scan();
   }
 
-  if(selector){
+  if (selector) {
     console.log("Found CSS selector of type " + selector.attribute + " and value " + selector.value);
```
Comment (pranaygp) [inline server/src/core/findSelector.ts]:
```javascript
console.log(`Found CSS selector of type ${selector.attribute} and value ${selector.value}`);
```

File: server/src/core/findSelector.ts [hunk 5]
```diff
@@ -47,21 +51,20 @@ export default function findSelector(document: TextDocument, position: Position)
         break;
       case TokenType.AttributeValue:
         if (attribute === 'class' || attribute === 'id') {
-          if (htmlScanner.getTokenText().slice(1, -1).split(' ').indexOf(selectorWord) > -1){
-            selector = { attribute, value: selectorWord}
+          if (htmlScanner.getTokenText().slice(1, -1).split(' ').indexOf(selectorWord) > -1) {
+            selector = { attribute, value: selectorWord }
```
Comment (pranaygp) [inline server/src/core/findSelector.ts]: Add semi

File: server/src/core/findSelector.ts [hunk 6]
```diff
@@ -22,11 +22,15 @@ export default function findSelector(document: TextDocument, position: Position)
   let start = offset;
   let end = offset;
 
-  while (start > 0 && text.charAt(start - 1) !== ' ' && text.charAt(start - 1) !== '\'' && text.charAt(start - 1) !== '"' && text.charAt(start - 1) !== '\n' && text.charAt(start - 1) !== '<')
-    start -= 1
+  let delimeters = [' ', '\'', '"', '\n', '<']
 
-  while (end < text.length && text.charAt(end) !== ' ' && text.charAt(end) !== '\'' && text.charAt(end) !== '"' && text.charAt(end) !== '\n' && text.charAt(end) !== '>')
+  while (start > 0 && delimeters.indexOf(text.charAt(start - 1)) === -1) {
+    start -= 1
```
Comment (pranaygp) [inline server/src/core/findSelector.ts]: Add semi

File: server/src/core/findDefinition.ts [hunk 7]
```diff
@@ -13,7 +13,7 @@ let languageServices: { [id: string]: LanguageService } = {
 export function getLanguageService(document: TextDocument) {
 	let service = languageServices[document.languageId];
 	if (!service) {
-		console.log('Document type is ' + document.languageId + ', using css instead.');
+    console.log(`Document type is ${document.languageId}, using css instead.`)
```
Comment (pranaygp) [inline server/src/core/findDefinition.ts]: Add semi

File: server/src/core/findDefinition.ts [hunk 8]
```diff
@@ -51,15 +51,14 @@ export function findSymbols(selector: Selector, stylesheetMap: StylesheetMap): S
     .forEach(uri => {
       const { document, stylesheet } = stylesheetMap[uri];
       const symbols = getLanguageService(document).findDocumentSymbols(document, stylesheet);
-      console.log('Found ' + symbols.length + ' symbols in ' + uri);
-
+      console.log(`Found ${symbols.length} symbols in ${uri}`)
```
Comment (pranaygp) [inline server/src/core/findDefinition.ts]: Add semi

File: server/src/server.ts [hunk 9]
```diff
@@ -28,7 +28,7 @@ let activeLanguages: string[];
 // A list of file extensions to lookup for style definitions (defaults to .css, .scss and .less)
 let fileSearchExtensions: string[]
 
-documents.onDidOpen((event) => {
+let onFileOpen = (event: any) => {
```
Comment (pranaygp) [inline server/src/server.ts]:
```javascript
const onFileOpenOrChange = event => {
```

Also, don't use `any`. This event is already implicitly of type `TextDocumentChangeEvent` so there's no reason to make the type more generic when we already know what they type is

File: server/src/server.ts [hunk 10]
```diff
@@ -105,12 +93,15 @@ connection.onDefinition((textDocumentPositon: TextDocumentPositionParams): Defin
   const document = documents.get(documentIdentifier.uri);
 
   // Ignore defintiion requests from unsupported languages
-  if(activeLanguages.indexOf(document.languageId) === -1){
+  if (activeLanguages.indexOf(document.languageId) === -1) {
```
Comment (pranaygp) [inline server/src/server.ts]: I think you meant to get rid of this, and not duplicate the logic

**Pull request closed without merging**